src/i18n/Auth.js

```diff
@@ -273,11 +273,11 @@ export default {
   },
   pleaseEnterClientID: {
     zh: '请输入 Client ID',
-    en: 'Please enter Client ID',
+    en: 'Client ID is required',
```

Comment:
Does the required statement of all form items have to be changed to this?

Reply:
YES. I think this requried statement is better?